.github/workflows/build_targeted.yaml

```
@@ -98,6 +106,17 @@ jobs:
             type=ref,event=pr
             type=sha,format=long,suffix=-${{ env.NAMESPACE }}
 
+      - name: Install 1Password CLI
+        if: ${{ inputs.op_env_file }}
+        uses: 1password/install-cli-action@v1
+
+      - name: 'Create env file'
+        if: ${{ inputs.op_env_file }}
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+        run: |
+          op document get "${{ secrets.op_env_file }}" --out-file=${{ secrets.op_env_file_out }}
```

```suggestion
          op document get "${{ inputs.op_env_file }}" --out-file=${{ inputs.op_env_file_out }}
```

these vars are not in secrets?

also, this could be useful:
https://github.com/FirstStreet/ci-workflow/compare/master...1pass

also, I think `.github/workflows/targeted.yml` needs the same change